cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/stores/store.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+class Store {
```

Comment:
We should probably move this to utilities at some point. I could see us reusing this in the future.

Reply:
Cool I'll sprinkle some `TODO`s throughout the code for future PRs.